webpack.config.js

```diff
@@ -165,18 +166,21 @@ if (!__TEST__) {
 // ------------------------------------
 // Rules
 // ------------------------------------
+const jsLoaders = [{
+  loader: 'babel-loader',
+  options: {
+    cacheDirectory: true,
+  },
+}]
+if (__DEV__) jsLoaders.unshift('react-hot-loader/webpack')
```

Comment:
I can find this in docs, but it [works](http://andrewhfarmer.com/webpack-2-loader-variations/#3-rule-with-multiple-loaders-no-options).